cdk/lib/acquisition-events-api.ts

```diff
@@ -1,23 +1,135 @@
 import { join } from "path";
+import { GuApiLambda } from "@guardian/cdk";
 import type { GuStackProps } from "@guardian/cdk/lib/constructs/core";
 import { GuStack } from "@guardian/cdk/lib/constructs/core";
 import type { App } from "aws-cdk-lib";
+import { Duration } from "aws-cdk-lib";
+import { CfnBasePathMapping, CfnDomainName } from "aws-cdk-lib/aws-apigateway";
+import {Effect, Policy, PolicyStatement} from "aws-cdk-lib/aws-iam";
+import { Runtime } from "aws-cdk-lib/aws-lambda";
+import {CfnRecordSet} from "aws-cdk-lib/aws-route53";
 import { CfnInclude } from "aws-cdk-lib/cloudformation-include";
```

Comment:
```suggestion
```

Reply:
I think it's better to leave it here till I remove the existing API Gateway Instance in next phase